src/AWS.Deploy.CLI/ServerMode/Tasks/DeployRecommendationTask.cs

```diff
@@ -30,6 +32,14 @@ public async Task Execute()
             await _orchestrator.DeployRecommendation(_cloudApplication, _selectedRecommendation);
         }
 
+        public async Task<string> CreateCDKDeploymentProject(CdkProjectHandler cdkProjectHandler)
+        {
+            await CreateDeploymentBundle();
+            var cdkProject = await cdkProjectHandler.ConfigureCdkProject(_orchestratorSession, _cloudApplication, _selectedRecommendation);
+            await cdkProjectHandler.PerformCdkDiff(cdkProject);
```

Comment:
A method called `CreateCDKDeploymentProject` should also need to run `PerformCdkDiff`. I understand why you are doing it but we should be clear with our names and intents.

Reply:
Done